### ISSUE TITLE
Can't compile on Windows 10

### ISSUE THREAD

**Issue by lesleyrs** [hexops/mach#716]
```
libs\mach\libs\core\src\main.zig:1:1: error: file exists in multiple modules
libs\mach\libs\core\src\main.zig:1:1: note: root of module root.app.mach.core
libs\mach\libs\core\src\main.zig:1:1: note: root of module root
libs\mach\libs\gpu\src\main.zig:1:1: error: file exists in multiple modules
libs\mach\libs\gpu\src\main.zig:1:1: note: root of module root.app.gpu
libs\mach\libs\gpu\src\main.zig:1:1: note: root of module root.gpu
libs\mach\libs\gpu\src\interface.zig:1:21: note: imported from module root.gpu
const gpu = @import("main.zig");
                    ^~~~~~~~~~
libs\mach\libs\gpu\src\dawn_impl.zig:1:21: note: imported from module root.gpu
const gpu = @import("main.zig");
                    ^~~~~~~~~~
```

Just used `zig build` and it didn't work, I see in the known issues some windows errors about symlinks but this says something different?

**Comment by dweiller** [comment]
The issue seems to be a change in the Zig build system that means the current design of Mach's `build.zig` files doesn't work with versions of Zig since the new module CLI syntax got merged (commit 7f6911b3fe in the Zig repo). I started working on updating the `build.zig` files, but it's a pretty invasive change and haven't finished, and I'm not sure how Stephen would want things restructured.

**Comment by slimsag** [comment]
Download links for the older version (courtesy of Andrew Kelley :) many thanks to him for helping me dig these up):

https://ziglang.org/builds/zig-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-linux-x86_64-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-bootstrap-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-macos-aarch64-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-linux-aarch64-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-linux-riscv64-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-linux-powerpc-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-linux-powerpc64le-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-macos-x86_64-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-linux-x86-0.11.0-dev.1711+dc1f50e50.tar.xz
https://ziglang.org/builds/zig-windows-aarch64-0.11.0-dev.1711+dc1f50e50.zip
https://ziglang.org/builds/zig-windows-x86_64-0.11.0-dev.1711+dc1f50e50.zip
https://ziglang.org/builds/zig-windows-x86-0.11.0-dev.1711+dc1f50e50.zip

That version should work in theory, I'll get us fixed up to the latest version very soon.
